### PR TITLE
Week2  Abuzer

### PR DESCRIPTION
**Pull request by abuz33**

### CODE REVIEW

File: Week2/js-exercises/project/app.js [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+}
+
+
+function decreaseLength() {
+  if (!countdown) {
+    if (parseInt(sessionLengthDisplay.textContent) > 1) {
+      sessionLengthDisplay.textContent = parseInt(sessionLengthDisplay.textContent) - 1;
+      minuteConverter(parseInt(sessionLengthDisplay.textContent));
+    } else {
+      alert('It cannot be less than 1');
```
Comment (wilgert) [inline Week2/js-exercises/project/app.js]: Have you tried setting the timer to one. Did you notice what happens if you accidentally click one time to many?
I think it is not very user friendly to reset the timer to 25 when the user makes one click to many!

File: Week2/js-exercises/project/app.js [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+
+
+function timeDisplay(seconds) {
+  let minute = Math.floor(seconds / 60);
+  let second = Math.floor(seconds % 60);
+  return `${minute}:${second < 10 ? 0 : ''}${second}`
+}
+
+
+function minuteConverter(minutes) {
```
Comment (wilgert) [inline Week2/js-exercises/project/app.js]: I think the name of this function is not really clear. The name implies that it is converting a minute to something else and returning it. But actually it is updating the display. Try to come up with a better name for this function.

File: Week2/js-exercises/project/app.js [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+const upArrow = document.querySelector('.up');
+const play = document.querySelector('.play');
+const pause = document.querySelector('.pause');
+
+const audioFinish = document.getElementById('finish');
+
+let sessionLengthDisplay = document.querySelector('.length');
+let timeLeftDisplay = document.querySelector('.timer');
+
+let alarm = alert('Timer is Runnig');
```
[the remark this answers is not in the document]
Comment (abuz33) [reply Week2/js-exercises/project/app.js]: Well, I created an alert inside of each function where it needed. And there wasn't any issue with that. Then I decide that it is repeating code. I did like that. I couldn't find solutions but I will return to my first solution ASAP.

File: Week2/js-exercises/project/app.js (hunk 1 above)
Comment (abuz33) [reply Week2/js-exercises/project/app.js]: Yeah. I did it on purpose. But I will make it stay on the same value.

**Comment by abuz33** [comment]
I made the changes.

File: Week2/js-exercises/project/app.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (wilgert) [reply Week2/js-exercises/project/app.js]: What you could do to reduce duplication is turn `alarm` in to a function that you call everywhere you need it.

[the remark this answers is not in the document]
Comment (abuz33) [reply Week2/js-exercises/project/app.js]: What you mean I understood wilgert. Late is better than never. I have created showAlert function. I did as you said.

**Comment by abuz33** [comment]
Thank you Andrej.

**Pull request closed without merging**